sycl/unittests/Extensions/CommandGraph.cpp

```diff
@@ -391,9 +391,19 @@ bool depthSearchSuccessorCheck(
 
 /// Submits four kernels with diamond dependency to the queue Q
 /// @param Q Queue to submit nodes to.
-void runKernels(queue Q) {
-  auto NodeA = Q.submit(
-      [&](sycl::handler &cgh) { cgh.single_task<TestKernel<>>([]() {}); });
+/// @param Dep Events to add as previous dependencies to the node group
+/// @return handler of the event associated to the last kernel submitted
```

Comment:
```suggestion
/// @return The event associated with the last kernel submitted
```
Nitpick but this read a little confusingly to me.